lib/begin.go

```diff
@@ -179,6 +190,9 @@ func (a *ACBuild) beginFromImage(start string, insecure bool) error {
 
 	err = reg.Fetch(app.Name, labels, 0, false)
 	if err != nil {
+		if _, ok := err.(*url.Error); ok {
+			return fmt.Errorf("unknown host when fetching image, local file paths must start with '/' or '.'")
```

Comment:
That didn't appear to be the error returned when I did something like `acbuild begin alpine-latest-linux-amd64.aci`.

I got `*url.Error` from a `fmt.Printf("%s\n", reflect.TypeOf(error).String())` just before the check.